client/src/BitTagPills.tsx

```diff
@@ -22,11 +23,11 @@ const BitTagPills: AppFunctionComponent<InputProps, NOOP> = (props) => {
     thunkSetLabels,
   } = props;
   return (
-    <div className="bit-tag-pills">
+    <div className={styles['bit-tag-pills']}>
```

Comment:
i'm not sure what the best practice is here, but in the sass if we name the class bitTagPills i think we might just be able to say `styles.bitTagPills` and have compile time checking? fine with leaving as-is too. there's probably a better way to do this idk.

Reply:
I think that is basically the choice AFAICT. Probably the camelcase is better for type checking. Too lazy to fix this now so I'll follow up.